src/main/java/org/osc/manager/ism/entities/PolicyElements.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+
+
+
+public class PolicyElements {
```

Comment:
Rename PolicyElements to PolicyEntity. We can use the suffix *Entity for all the classes inside this package for consistency. 
Also I think that we will no longer need the classes inside the package ism.model. We can probably just have the *.entities package now to simplify and just make sure those classes implement the *Element interfaces. In other words, delete PolicyListElement and make this class implement the SDK interface ManagerPolicyElement. @arvindn05  do you agree?